assets/newsletter-section.css

```
@@ -2,6 +2,49 @@
   padding-right: calc(4rem / var(--font-body-scale));
   padding-left: calc(4rem / var(--font-body-scale));
 }
+.newsletter__wrapper-new {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: 5rem;
+}
+.form-container {
+  display: flex;
+  height: 100%;
+  margin: 0 !important;
+}
+.input-container {
+}
+.field__input-new {
```

```suggestion
.field__input{
```
Same as before, let's remove all of the `-new` and modify what we need it to be.